exercises/farewell-workflow/solution/Workflow/TranslateActivities.cs

```diff
@@ -0,0 +1,23 @@
+// Translate Activities
+namespace Temporalio.Farewell.Workflow;
+
+using Temporalio.Activities;
+
+public class TranslateActivities
+{
+    private static readonly HttpClient client = new();
```

Comment:
There's an opportunity here to use non-static and DI if you want to inject the HTTP client in the constructor, but don't have to

Reply:
Bump. This actually may be more important now that I have checked and `HttpClient` is `IDisposable`. I suspect when you turn on analyzers, you will get a warning.